src/services/twilio-api/twilio_accounts_v1.json

```diff
@@ -4,19 +4,35 @@
       "accounts.v1.auth_token_promotion": {
         "properties": {
           "account_sid": {
-            "$ref": "#/components/schemas/sid_AC"
+            "description": "The SID of the Account that the secondary Auth Token was created for",
```

Comment:
I am trying to understand these changes. Isn't this undoing Elise's custom type changes? or am I mistaken?

Reply:
These changes in YPS will rollback the changes made to move the property schema definitions to a global component schema but it will not remove custom types from the library. We could not keep the property schema definitions as global components and include the description for properties because it would not conform to OpenAPI standards. In order to conform to OpenAPI standards we have to use the format below to include descriptions for the properties.
```
"account_sid": {
            "description": "The SID of the Account that the secondary Auth Token was created for",
            "maxLength": 34,
            "minLength": 34,
            "nullable": true,
            "pattern": "^AC[0-9a-fA-F]{32}$",
            "type": "string"
          },
```